src/store/audio.ts

```diff
@@ -706,17 +793,50 @@ export const audioStore = createPartialStore<AudioStoreTypes>({
       ),
   },
 
-  VALID_MOPHING_INFO: {
-    getter: (_, getters) => (audioItem: AudioItem) => {
-      if (
-        audioItem.morphingInfo == undefined ||
-        audioItem.engineId == undefined
-      )
+  /**
+   * 引数の2つのVoiceがモーフィング可能かどうか返す。
+   * strictCacheがtrueのときキャッシュが存在しない場合は不可とされる。
+   */
+  IS_A_VALID_MOPHING_PAIR: {
+    getter:
+      (state, getters) =>
+      (baseVoice, targetVoice, strictCache = false) => {
+        if (baseVoice.engineId !== targetVoice.engineId) {
+          return false;
+        }
+        const engineId = baseVoice.engineId;
+        if (!getters.MORPHING_SUPPORTED_ENGINES.includes(engineId)) {
+          return false;
+        }
+        const morphableTargets =
+          state.morphableTargetsInfo[engineId][baseVoice.styleId];
+        return morphableTargets === undefined
+          ? !strictCache
+          : morphableTargets[targetVoice.styleId]?.isMorphable;
```

Comment:
`morphableTargets[targetVoice.styleId]?.isMorphable`のところ、`undefined | boolean`が返ってそうですね 😇 
意図をわかりやすくするとこうですかね（あまり変わってないかも）
```suggestion
        if (morphableTargets == undefined) { return !strictCache; }
        return morphableTargets[targetVoice.styleId]?.isMorphable == true
```